server/webapp/WEB-INF/rails/spec/controllers/api_v1/admin/plugin_settings_controller_spec.rb

```diff
@@ -190,19 +190,35 @@
           ])))
       end
 
-      it 'should deserialize plugin settings from given object' do
+      it 'should deserialize plugin settings from given object with sane defaults for boolean and null' do
```

Comment:
Consider moving this up to plugin_configuration_property_representer_spec as plugin_configuration_property_representer.rb is shared among representers used for scm, package-repo, auth-config and plugin settings API?

Reply:
Yes, makes sense. I'll do that.

Reply:
How about this?

Reply:
LGTM